### DISCUSSION TITLE
TranslucentTB

### DISCUSSION THREAD

**Discussion by u-deep** [AutoDarkMode/Windows-Auto-Night-Mode#830] [Custom scripts]
here i have done my personal script to set different TB behaviour in dark/light mode

https://github.com/u-deep/AutoDarkMode-scripts-for-TranslucentTB

**Comment by connor-129** [comment]
I had to change the filepath strings to raw strings, but now works great. Thanks!

Eg:
`light_settings_source = r"C:/Users/****/AppData/Roaming/AutoDarkMode/lightsettings.json"`
`destination_folder = r"C:\Users\****\AppData\Local\Packages\28017CharlesMilette.TranslucentTB_v826wp6bftszj\RoamingState"`
